app/scripts/templates/visits/item.html

```diff
@@ -1,4 +1,12 @@
-<div class="visit">
+<div class="image" style="background-image: url({{extractedImageUrl}});"></div>
```

Comment:
Yeah we're doing this because we're using CSS to "crop" the images using background cover.